### PR TITLE
deathTimer added

### PR DESCRIPTION
**Pull request by twolfe2**
I created this plugin for myself so that I could be reminded that death is always near. This plugin helps to keep me motivated and I hope it will help keep others motivated too. 

This plugin will allow a user to input the estimated year they will die and then it will provide them with a countdown to that year.

### CODE REVIEW

File: Lifestyle/deathTimer.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# <bitbar.desc>Counts down how approximately how many days you have left to live</bitbar.desc>
+# <bitbar.image>http://imgur.com/2KjZaFb</bitbar.image>
+# <bitbar.dependencies>python</bitbar.dependencies>
+
+
+
+from datetime import datetime
+from datetime import timedelta
+from datetime import datetime
+from datetime import timedelta
```
Comment (keithamus) [inline Lifestyle/deathTimer.py]: Looks like you're importing things twice here.

Comment (twolfe2) [inline Lifestyle/deathTimer.py]: Fixed, thanks.

**Comment by keithamus** [comment]
Oh, also I just noticed the file is called `deathTimer.py` - but should probably be `deathTimer.1s.py` or something?

**Comment by twolfe2** [comment]
Since right now it will be displaying the approximate number of days remaining I will have it update every 5 minutes. In a future update I might have it display days, hours, minutes, and seconds remaining.

**Comment by keithamus** [comment]
What a wonderfully somber plugin btw :smile:.

**Pull request merged**